apko-build/action.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Build image with apko'
+description: |
+  This action build an OCI image with apko, given a config file
+  and tag to use.
+
+inputs:
+  config:
+    description: |
+      The config file to use for building the image.
+    default: .apko.yaml
+
+  tag:
+    description: |
+      The tag to use for publishing the image.
+    required: true
+
+  repository_owner:
+    description: |
+      The repository owner's GitHub username.
+    default: ${{ github.repository_owner }}
+
+  token:
+    description: |
+      The repository owner's GitHub token.
+    default: ${{ github.token }}
```

Comment:
I think if we just tell folks that have to have auth configured, and that we respect the Docker keyring by default we can kill this and have folks use the login plugin.

Reply:
Doesn't work, because the docker credentials don't get passed through to the apko image, unfortunately...